source/stylesheets/scss/furatto/_pseudo_selectors.scss

```diff
@@ -0,0 +1,7 @@
+// Aligns the content vertically speaking
+%vertical-alignment {
+  position: relative;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Properties should be ordered margin, position, top